src/examples/HelloSPTP.cpp

```diff
@@ -139,7 +139,11 @@ void run() {
       const size_t timeTotal = (size_t)floor(tAll.getElapsed());
       cout << "Total elapsed time = " << timeTotal << " seconds" << endl;
       #ifdef NDEBUG
-        const size_t CI_avg_time = (size_t)floor(7*Timer::getSpeed()); //sec
+	    #ifdef _MSC_VER
+          const size_t CI_avg_time = (size_t)floor(14*Timer::getSpeed()); //sec
+		#else
+          const size_t CI_avg_time = (size_t)floor(7*Timer::getSpeed()); //sec
```

Comment:
👍 nice that you made 2 versions of the check, still keeping the unix test useful